src/harness/testcases/WINNF_FT_S_HBT_testcase.py

```diff
@@ -0,0 +1,155 @@
+    request = {'registrationRequest': devices}
+    response = self._sas.Registration(request)['registrationResponse']
+    cbsd_ids = []
+    for resp in response:
+        cbsd_ids.append(resp['cbsdId'])
+    cbsd_ids.sort(key=lambda cbsdId: cbsdId)
+    del request, response    
+    # create and send grant requests
+    grant_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_c['cbsdId'] = cbsd_ids[0]
+    grant_d = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_d['cbsdId'] = cbsd_ids[1]
+    grant_d['operationParam']['operationFrequencyRange']['lowFrequency'] = 3560
+    grant_d['operationParam']['operationFrequencyRange']['highFrequency'] = 3570
+    grant_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_e['cbsdId'] = cbsd_ids[2]
+    grant_e['operationParam']['operationFrequencyRange']['lowFrequency'] = 3560
+    grant_e['operationParam']['operationFrequencyRange']['highFrequency'] = 3570
+    request = {'grantRequest': [grant_c, grant_d, grant_e]}
+    # send grant request and get response
+    response = self._sas.Grant(request)['grantResponse']
+    self.assertEqual(len(response), 3)       
+    grant_ids = (response[0]['grantId'], response[1]['grantId'], response[2]['grantId'])
+    del request, response
+    # First Heartbeat
+    first_HB_response = self.sendAndAssertSuccessfulHeartBeat(cbsd_ids, grant_ids, [None, None, None])
+    transmitExpireTime = (first_HB_response[0]['transmitExpireTime'], first_HB_response[1]['transmitExpireTime'], first_HB_response[2]['transmitExpireTime'])
+    # STEP 2
+    # load and inject FSS data
+    fss_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'fss_record_c.json')))
+    # Inject Incumbent Activity with Overlapping Frequency of CBSD
+    fss_c['deploymentParam']['operationParam']['operationFrequencyRange']['lowFrequency'] = \
+                             grant_c['operationParam']['operationFrequencyRange']['lowFrequency']
+    fss_c['deploymentParam']['operationParam']['operationFrequencyRange']['highFrequency'] = \
+                             grant_c['operationParam']['operationFrequencyRange']['highFrequency']
+    
+    self._sas_admin.InjectFss({'record': fss_c})   
+    # load and inject GWPZ data
+    gwpz_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'gwpz_record_c.json')))
+    # Inject Incumbent Activity with Overlapping Frequency of CBSD
+    gwpz_c['deploymentParam']['operationParam']['operationFrequencyRange']['lowFrequency'] = \
+                             grant_c['operationParam']['operationFrequencyRange']['lowFrequency']
+    gwpz_c['deploymentParam']['operationParam']['operationFrequencyRange']['highFrequency'] = \
+                             grant_c['operationParam']['operationFrequencyRange']['highFrequency']
+    
+    self._sas_admin.InjectWisp({'record': gwpz_c})      
```

Comment:
What about the 'zone' key? See https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/blob/master/src/harness/sas_interface.py#L288

Reply:
the issues of the comments related to the record files are done, please verify them in the latest change.
for the zone, I added it a polygon(rectangle) around the center, but we should modify it later according to realistic GWPZ shape data, is that ok ?
thank you

Reply:
This sounds good!